pkg/restapi/vc/operation/operations.go

```diff
@@ -55,6 +55,8 @@ const (
 	vcStatusEndpoint               = vcStatus + "/{id}"
 	credentialsBasePath            = "/credentials"
 	issueCredentialPath            = credentialsBasePath + "/issueCredential"
+	kmsBasePath                    = "/kms"
```

Comment:
nit: not sure this name is appropriate given that this system is not a KMS

Reply:
well, its not domain name - its subpath (operation on kms). Ideally, full path would look like "https://issuer.example.com/kms/generatekeypair".

Reply:
a path named "kms" would denote a KMS resource.

But I'm not going to be picky about this one - we can easily refactor later.

Reply:
Agree on the refactoring part.